packages/financial-templates-lib/src/price-feed/CreatePriceFeed.js

```diff
@@ -62,7 +64,7 @@ async function createPriceFeed(logger, web3, networker, getTime, config) {
       getTime,
       config.minTimeBetweenUpdates,
       config.invertPrice, // Not checked in config because this parameter just defaults to false.
-      config.decimals, // This defaults to 18 unless supplied by user
+      config.priceFeedDecimals, // This defaults to 18 unless supplied by user
```

Comment:
This was an error for the DomFi price feed. The file `DefaultPriceFeedConfig` sets the `priceFeedDecimals` param not `decimals`:

```
// Pull in the number of decimals for each identifier from the common getPrecisionForIdentifier. This is used within the
// Voterdapp and ensures that price feeds are consistently scaled through the UMA ecosystem.
Object.keys(defaultConfigs).forEach(identifierName => {
  defaultConfigs[identifierName].priceFeedDecimals = getPrecisionForIdentifier(identifierName);
});
```

Luckily this is not an issue in prod since the default decimal is 18 which the DomFi EMP's use